src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+@use 'variables' as v;
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+
+    &__title {
+      color: #34568b;
+    }
```

Comment:
The '&__title' selector is nested under the ':hover' pseudo-class, which means the title color change will only apply when the card is hovered. If the intention is to change the title color on hover, this is correct. Otherwise, consider moving it outside the hover block.